File: clients/async-scala/src/main/scala/io/swagger/client/model/OrderBook.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class OrderBook (
  symbol: String,
level: Number,
bidSize: Option[Number],
bidPrice: Option[Double],
askPrice: Option[Double],
askSize: Option[Number],
timestamp: Option[Date]
)

File: clients/typescript-angular/API/Client/OrderBook.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface OrderBook {

        symbol: string;

        level: number;

        bidSize?: number;

        bidPrice?: number;

        askPrice?: number;

        askSize?: number;

        timestamp?: date;
    }

}
